### DISCUSSION TITLE
Would it be possible to move `per_page` for pagination into an option?

### DISCUSSION THREAD

**Discussion by 6r33z3** [plausible/analytics#1507] [Anything Else]
Upon testing with the new v1.4.x, we noticed all sites are being paginated as per https://github.com/plausible/analytics/pull/994/commits/f2b00cf134526e1e3a1c0a20cd2daf33b2bc57df

This is definitely a step into the right direction, however in our case, as sysadmin, it would be more convenient to be able to have a glance of all sites we have (much more than 18) and having this option customizable, e.g. either set to 0 to disable, or a custom value fetchable from settings would be great.

**Comment by ukutaht** [comment]
Hey. That's reasonable. For now you should be able to use `?per_page=100` query parameter on the sites page to get a longer list
